DNN Platform/Library/Services/Exceptions/SearchException.cs

```diff
@@ -23,19 +18,10 @@ public SearchException()
         /// <param name="message"></param>
         /// <param name="inner"></param>
         /// <param name="searchItem"></param>
-        public SearchException(string message, Exception inner, SearchItemInfo searchItem)

```

Comment:
Given that we don't have anything I'm not sure.  My only reason for leaving it, was it is used by the indexer process to capture a known issue rather than a global `Exception` catch

Reply:
I don't have a problem keeping this since it's used